dotCMS/src/integration-test/java/com/dotcms/datagen/FieldDataGen.java

```diff
@@ -8,16 +8,14 @@
 
 public class FieldDataGen extends AbstractDataGen<Field> {
 
-    private final long currentTime = System.currentTimeMillis();
-
-    private String name = "testFieldName" + currentTime;
+    private String name;
     private Class type = TextField.class;
     private boolean required = Boolean.FALSE;
-    private String velocityVarName = "testFieldVarname" + currentTime;
+    private String velocityVarName;
     private int sortOrder = 1;
-    private String values = "testValue" + currentTime;
-    private String hint = "testHint" + currentTime;
-    private String defaultValue = "testDefaultValue" + currentTime;
+    private String values;
+    private String hint;
```

Comment:
Codacy found an issue: [Field hint has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6326102)